src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/data/producer/SamplePaymentMessageData.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer;
+
+import com.google.common.collect.ImmutableList;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentInfo;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+public final class SamplePaymentMessageData {
+
+    public static final String ENVELOPE_ID = "99999ZS";
+    public static final String JURSIDICTION = "BULKSCAN";
+    public static final String PO_BOX = "BULKSCAN_PO_BOX";
+    public static final String DCN_1 = "xxxyyyzzz";
+    public static final String DCN_2 = "ABCDDDDDD";
+
+    private SamplePaymentMessageData() {
+    }
+
+    public static String paymentMessageJson(String ccdCaseNumber, boolean isExceptionRecord) {
+        try {
+            return new JSONObject()
+                .put("envelope_id", ENVELOPE_ID)
+                .put("ccd_case_number", ccdCaseNumber)
+                .put("is_exception_record", isExceptionRecord)
+                .put("po_box", PO_BOX)
+                .put("jurisdiction", JURSIDICTION)
+                .put("payments", new JSONArray()
+                    .put(new JSONObject().put("document_control_number", DCN_1))
+                    .put(new JSONObject().put("document_control_number", DCN_2))
+                )
+                .toString();
+        } catch (Exception e) {
+            throw new RuntimeException("Could not make paymentMessageJson", e);
```

Comment:
can just throws in method signature no need for try catch

Reply:
I do not see any drawback but I changed.